bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/handler/DaikinAcUnitHandler.java

```diff
@@ -312,6 +337,34 @@ private void changeMode(Mode mode) throws DaikinCommunicationException {
         webTargets.setControlInfo(info);
     }
 
+    private void changeHomekitMode(String homekitmode) throws DaikinCommunicationException {
+        ThingTypeUID thingTypeUID = thing.getThingTypeUID();
+        if (homekitmode.equals("off")) {
+           changePower(false);
+        } else {
+           changePower(true);
+           if (homekitmode.equals("on")) {
+              if (thingTypeUID.equals(DaikinBindingConstants.THING_TYPE_AC_UNIT)) {
+                changeMode(Mode.valueOf(((String) "AUTO").toString()));
```

Comment:
Just pass the enum values directly. It's all static content here, so no need to get the values from String.
```suggestion
                changeMode(Mode.AUTO);
```

Comment applies to all use cases.